pisound-btn/pisound-btn.c

```diff
@@ -1121,6 +1162,10 @@ int main(int argc, char **argv, char **envp)
 				return 1;
 			}
 		}
+		else if (strcmp(argv[i], "--active_low") == 0)
```

Comment:
Maybe `--active-low`? To be consistent with `--click-count-limit`, where - was already used for word separator instead of _.